.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+- package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+  allow:
+  - dependency-type: "direct"
+  groups:
+    k8s:
+      patterns:
+      - "k8s.io/*"
+      - "sigs.k8s.io/*"
+    onsi:
+      patterns:
+      - "github.com/onsi/*"
```

Comment:
question: for dependencies not included here, will they also be automatically bumped?

Reply:
yes, other dependencies will also be bumped. this just says to bump the group together if there are updates for multiple dependencies in the group.

Reply:
Thanks for the clarification.